modules/fbs/includes/fbs.user.inc

```diff
@@ -110,3 +110,15 @@ function fbs_user_update_pincode($account, $new_pincode) {
   $result['creds']['library_card_number'] = $patron_info['library_card_number'];
   return $result;
 }
+
+/**
+ * Implements clear_cache().
+ *
+ * This hook is only called from ding_dibs after a payment have been accepted.
+ *
+ * @param stdClass $account
```

Comment:
This should be "object" as identified by scrutinizer. It must be "object" as that is a instance of a class.